commit 8493c52be3cc71b110bc5d08d4531032f5a282b8 (refs/changes/89/1789/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-13T12:30:50-07:00 (2 years, 6 months ago)
    
    joyent/node-triton#199 cli-networks.test.js crash if there are no non-public networks
    Reviewed by: Jason King <jason.king@joyent.com>
    Approved by: Jason King <jason.king@joyent.com>

diff --git a/test/integration/cli-networks.test.js b/test/integration/cli-networks.test.js
index 1d822dd..dad5ef8 100644
--- a/test/integration/cli-networks.test.js
+++ b/test/integration/cli-networks.test.js
@@ -71,10 +71,13 @@ test('triton networks', function (tt) {
         function (err, stdout, stderr) {
             if (h.ifErr(t, err))
                 return t.end();
-            var results = stdout.trim().split('\n');
-            results.forEach(function (result) {
-                t.equal(false, common.boolFromString(result, null, 'public'));
-            });
+            if (stdout.trim()) {
+                var results = stdout.trim().split('\n');
+                results.forEach(function (result) {
+                    t.equal(false,
+                        common.boolFromString(result, null, 'public'));
+                });
+            }
             t.end();
         });
     });
@@ -84,10 +87,13 @@ test('triton networks', function (tt) {
         function (err, stdout, stderr) {
             if (h.ifErr(t, err))
                 return t.end();
-            var results = stdout.trim().split('\n');
-            results.forEach(function (result) {
-                t.equal(false, common.boolFromString(result, null, 'public'));
-            });
+            if (stdout.trim()) {
+                var results = stdout.trim().split('\n');
+                results.forEach(function (result) {
+                    t.equal(false,
+                        common.boolFromString(result, null, 'public'));
+                });
+            }
             t.end();
         });
     });
